goss-testing/scripts/check_clock_skew.sh

```diff
@@ -44,42 +44,42 @@ for node in $nodes; do
   pdsh_node_args="$pdsh_node_args -w $node"
 done
 
-for try in {1..3}; do
-  # Set/reset variables at the top of each attempt loop
-  cnt=0
-  exit_code=0
-  baseline=""
+# Set/reset variables at the top of each attempt loop
```

Comment:
This comment no longer makes sense given that the retry loop has been removed.

```suggestion
```

Reply:
Good catch.  Fixed.